payments/tests/factories.py

```diff
@@ -31,16 +26,9 @@ class Meta:
 
 
 class BerthPriceGroupFactory(factory.django.DjangoModelFactory):
-    name = factory.Faker("word")
```

Comment:
as we have discussed, you should probably leave this line in this Factory, otherwise it won't work